src/server/scripts/EasternKingdoms/ScarletMonastery/instance_scarlet_monastery.cpp

```diff
@@ -455,7 +498,10 @@ class npc_mograine : public CreatureScript
 
         void KilledUnit(Unit* /*victim*/) override
         {
-            Talk(SAY_MO_KILL);
+            // Don't yell on death during Ashbringer event
+            if (!SayAshbringer) {
```

Comment:
curlybracket isnt needed

Reply:
Hmm, am I missing something in my local devenv that the files are not getting automatically formatted to comply with the project's code styles? If so, could you please point me to the right documentation to set it up?

Reply:
Okay, I found out that this project had had such tools and no longer. I ended up setting up `clang-format` for myself locally and fixing the formatting with it & by hand :)